### PR TITLE
f38172 Global fix code.

### PR DESCRIPTION
**Pull request by BasilSnowman**
Fix checking domain, fix replace block.

### CODE REVIEW

File: src/Plugin/Block/DomainMenuAccessMenuBlock.php [hunk 1]
```diff
@@ -18,6 +22,45 @@
  */
 class DomainMenuAccessMenuBlock extends SystemMenuBlock implements ContainerFactoryPluginInterface {
 
+  protected $menuStorage;
```
Comment (andypost) [inline src/Plugin/Block/DomainMenuAccessMenuBlock.php]: needs doc block

File: src/Plugin/Block/DomainMenuAccessMenuBlock.php [hunk 2]
```diff
@@ -18,6 +22,45 @@
...
+   * @param string $plugin_id
+   *   The plugin_id for the plugin instance.
+   * @param array $plugin_definition
+   *   The plugin implementation definition.
+   * @param \Drupal\Core\Menu\MenuLinkTreeInterface $menu_tree
+   *   The menu tree service.
+   * @param \Drupal\Core\Menu\MenuActiveTrailInterface $menu_active_trail
+   *   The active menu trail service.
+   * @param \Drupal\Core\Config\Entity\ConfigEntityStorageInterface $menu_storage
+   * @internal param \Drupal\system\MenuInterface $menuStorage The menu storage.*   The menu storage.
```
Comment (andypost) [inline src/Plugin/Block/DomainMenuAccessMenuBlock.php]: rm please

File: src/Plugin/Block/DomainMenuAccessMenuBlock.php [hunk 3]
```diff
@@ -64,9 +107,10 @@ public function build() {
    *   Config enabled.
    */
   protected function isDomainRestricted($menu_name) {
-    $config = \Drupal::config('domain_access.settings')->get('menu_enabled');
+    /** @var \Drupal\system\Entity\Menu $menu */
```
Comment (andypost) [inline src/Plugin/Block/DomainMenuAccessMenuBlock.php]: `\Drupal\system\MenuInterface`

File: src/Plugin/Block/DomainMenuAccessMenuBlock.php (hunk 2 above)
Comment (andypost) [reply src/Plugin/Block/DomainMenuAccessMenuBlock.php]: description for argument The menu storage.

**Review by andypost — CHANGES_REQUESTED** [review]
Needs cleanup

**Comment by BasilSnowman** [comment]
fixed

**Review by andypost — APPROVED** [review]
Great!

**Pull request merged**